eventually/tests/unittests/authentication/testview.py

```diff
@@ -95,3 +125,154 @@ def test_activation_bad_request(self):
+
+
+    def test_put_password(self):
+        """Positive test on password change"""
+
+        user = json.dumps({'email':'mailbox1@mail.co', 'password':'12345678'})
+        response = self.client.post(reverse('login_user'), user, content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+        user_password = json.dumps({"password": "1Aa"})
+        password_change_response = self.client.put('/api/v1/user/10/', user_password)
+        self.assertEqual(password_change_response.status_code, 200)
+
+
+    def test_put_negative_password(self):
+        """ Negative test on change password """
+
+        user = json.dumps({'email':'mailbox1@mail.co', 'password':'12345678'})
+        response = self.client.post(reverse('login_user'), user, content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+        user_password = json.dumps({"password": "1a"})
+        password_change_response = self.client.put('/api/v1/user/10/', user_password)
+        self.assertEqual(password_change_response.status_code, 400)
+
+
+    def test_put_negative(self):
+        """ User change negative test """
+
+        user = json.dumps({'email':'mail@mail.co', 'password':'1234'})
+        response = self.client.post(reverse('login_user'), user, content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+        user = json.dumps({"first_name": "Jerry"})
+        user_change_response = self.client.put('/api/v1/user/100500/', user)
+        self.assertEqual(user_change_response.status_code, 404)
+
+
+    def test_delete(self):
+        """ Delete user positive test """
+
+        user = json.dumps({'email':'delete@mail.co', 'password':'123456'})
+        response = self.client.post(reverse('login_user'), user, content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+        delete_user_response = self.client.delete('/api/v1/user/1/')
+        self.assertEqual(delete_user_response.status_code, 200)
+
+
+    def test_delete_negative(self):
+        """ User delete negative test """
+        user = json.dumps({'email':'delete@mail.co', 'password':'123456'})
+        response = self.client.post(reverse('login_user'), user, content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+        delete_user_reponse = self.client.delete('/api/v1/user/100/')
+        self.assertEqual(delete_user_reponse.status_code, 403)
+
+        delete_user_reponse = self.client.delete('/api/v1/user/100500/')
+        self.assertEqual(delete_user_reponse.status_code, 404)
+
+
+    def test_login(self):
+        """ Positive user login test """
+
+        user = json.dumps({'email':'mail@mail.co', 'password':'1234'})
+        response = self.client.post(reverse('login_user'), user, content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+
+    def test_login_negative(self):
```

Comment:
maybe it would be better to divide this test cases?

Reply:
Divided already.